client/actions/editActions.js

```diff
@@ -1,12 +1,62 @@
 import keyMirror from 'client/lib/keyMirror'
 
 export const actions = keyMirror(
-  'EDIT'
+  'CHANGE_SAVED_STATUS',
+  'CHANGE_SECTION',
+  'DELETE_ARTICLE',
+  'PUBLISH_ARTICLE',
+  'SAVE_ARTICLE'
 )
 
-export const edit = (isEditing) => ({
-  type: actions.EDIT,
+export const changeSavedStatus = (isSaved) => ({
+  type: actions.CHANGE_SAVED_STATUS,
   payload: {
-    isEditing
+    isSaved
   }
 })
+
+export const changeSection = (activeSection) => ({
+  type: actions.CHANGE_SECTION,
+  payload: {
+    activeSection
+  }
+})
+
+export const deleteArticle = (article) => {
+  article.destroy({
+    success: () => {
+      article.trigger('finished')
+    }
+  })
```

Comment:
👍 Nice moving backbone event stuff in with the actions to keep things contained. It might be good to group these all together and add a comment above the section giving a bit of overview.